backend/src/main/java/com/cruru/email/controller/EmailController.java

```diff
@@ -45,4 +50,15 @@ public ResponseEntity<Void> verifyCode(@Valid @RequestBody VerifyCodeRequest req
         emailFacade.verifyCode(request);
         return ResponseEntity.ok().build();
     }
+
+    @GetMapping("/{clubId}/{applicantId}")
+    @ValidAuth
+    public ResponseEntity<EmailResponses> read(
+            @RequireAuth(targetDomain = Club.class) @PathVariable Long clubId,
+            @PathVariable Long applicantId,
```

Comment:
```suggestion
            @RequireAuth(targetDomain = Applicant.class) @PathVariable Long applicantId,
```

여기에도 인가가 적용되어야할 것 같습니다~~